pallets/phala/src/stakepool.rs

```diff
@@ -357,6 +357,10 @@ pub mod pallet {
 		WorkerExists,
 		/// The target worker is not in the pool
 		WorkerDoesNotExist,
+		/// The target miner is not in the 	`miner` storage
```

Comment:
Please only add new errors to the end to keep compatibility.

Reply:
done